src/open_inwoner/pdc/admin/category.py

```diff
@@ -189,12 +191,56 @@ class CategoryAdmin(
             },
         ),
     )
+    filter_horizontal = [
+        "access_groups",
+    ]
+
+    list_filter = [
+        "published",
+        "visible_for_anonymous",
+        "visible_for_companies",
+        "visible_for_citizens",
+        "access_groups",
+    ]
 
     # import-export
     import_template_name = "admin/category_import.html"
     resource_class = CategoryImportResource
     formats = [base_formats.XLSX, base_formats.CSV]
 
+    def get_fields(self, request, obj=None):
+        fields = list(super().get_fields(request, obj=obj))
+        if not request.user.has_perm("auth.change_group"):
+            try:
+                fields.remove("access_groups")
+            except ValueError:
+                pass
+        return fields
```

Comment:
Does this work currently? I tried to trigger this by visiting the detail page and I could still see the field:
![image](https://github.com/maykinmedia/open-inwoner/assets/29249171/e2dc8286-6888-4297-bb0f-4075b5c6ca0b)

The user group had these permissions:
![image](https://github.com/maykinmedia/open-inwoner/assets/29249171/b1ef0168-fae3-449b-a413-c369952ab0ca)

Reply:
I removed this, this was from before I managed to add the m2m field to the Group's admin and makes more sense there.